api-report/cell.api.md

```diff
@@ -37,7 +35,7 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>> impl
     constructor(id: string, runtime: IFluidDataStoreRuntime, attributes: IChannelAttributes);
     // @internal (undocumented)
     protected applyStashedOp(content: unknown): unknown;
-    static create(runtime: IFluidDataStoreRuntime, id?: string): SharedCell<any>;
```

Comment:
`any` is the default type parameter here, so there is no actual type-wise change to the API.